package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "eslint-plugin-unicorn",
 	"version": "42.0.0",
-	"description": "Various awesome ESLint rules",
+	"description": "More than 100 powerful ESLint rules",
```

Comment:
```suggestion
	"description": "World's mightiest ESLint rules",
```

Reply:
```suggestion
	"description": "ESLint rules that really whip the llama's ass",
```

Reply:
You are an old man!

Reply:
![image](https://user-images.githubusercontent.com/1402241/161367716-ff1326e3-f815-472a-a9a7-e5e88abc526a.gif)